threeML/classicMLE/joint_likelihood.py

```diff
@@ -781,7 +784,7 @@ def worker(start_index):
 
                 aidx, bidx = np.unravel_index(idx, cc.shape)
 
-                print(
+                log.warning(
```

Comment:
I wonder if these should be more than warnings.. this seems pretty critical?

Reply:
Hm. We could log them as errors, I guess. I would not want to abort execution though, for two reasons: 

1. In interactive analysis, the user could just call `fit()` again after seeing the warning, and if we bail out, they'll lose the current state/settings.
2. In some cases (e.g. calculating upper limits), I'm only interested in the likelihood profile, not the minimum. I think the profile is still useful even if the initial minimum was off (but please correct me if I'm wrong).

Reply:
Ah, good point!